.github/workflows/scripts/scanner-offline-bundle-version-matrix.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+#
+# Print the mapping of StackRox releases to scanner vulnerability bundle schema
+# versions. One entry per line, separated by whitespace.
+
+set -euo pipefail
+
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+# GITHUB_URL_PATTERN points to vulnerability version file for a given branch/tag.
+GITHUB_URL_PATTERN="https://raw.githubusercontent.com/stackrox/stackrox/%s/scanner/VULNERABILITY_VERSION"
+
+RELEASE_VERSION_FILE="scanner/updater/version/RELEASE_VERSION"
+
+[ -f $RELEASE_VERSION_FILE ] || {
+    echo "error: release version file does not exist: $RELEASE_VERSION_FILE"
+    exit 1
+}
+
+# version_object() prints a version object in the matrix.
```

Comment:
nit: first thought was, "what is a version object", perhaps:

```suggestion
# version_object() prints a JSON object that maps a rox version to supported releases.
```

Reply:
I did a variation of it.

be872c3370